mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -138,9 +158,11 @@ def _extract_run_id(artifact_uri):
         artifact_path = extract_and_normalize_path(artifact_uri)
         return artifact_path.split("/")[3]
 
-    def _call_endpoint(self, service, api, json_body):
+    def _call_endpoint(self, service, api, json_body, path_params=None):
```

Comment:
Can we add a docstring to this method while we're here, and can we document the parameters?